assets/styles/app/tom-select.css

```diff
@@ -0,0 +1,419 @@
+/**
+ * tom-select.css (v2.3.1)
+ * Copyright (c) contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at:
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+ * ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ *
+ * Modifications:
+ *   19 March 2024: Updated `.ts-wrapper.multi .ts-control > div.active` and `.ts-wrapper.multi .ts-control > div`
+ *   to round the item borders and changed the background color.
+ */
```

Comment:
I just want to draw attention to the License and Modifications section. If my understanding of the Apache License v2.0 is correct, this is all that's needed